modules/@apostrophecms/image/ui/apos/components/AposImageRelationshipEditor.vue

```diff
@@ -131,32 +131,40 @@ export default {
   ],
   props: {
     schema: {
+      type: Array,
+      default: null
+    },
+    widgetSchema: {
       type: Array,
       default: () => ([])
     },
-    modelValue: {
+    item: {
```

Comment:
I don't think we can change this. It should also comply with the regular `AposRelationshipEditor` component.

Reply:
rollbacked this